source/src/ble/cs_Stack.cpp

```diff
@@ -6,6 +6,7 @@
  */
 
 #include <algorithm>
+#include <ble/cs_BleCentral.h>
```

Comment:
Since a lot changed in this file anyway, could you maybe move the leftover documentation in the .cpp to the .h file? And perhaps remove the commented out code?

Reply:
We'll clean up the Stack later =]